### PR TITLE
fix(remix-dev): resolve asset entry full path to support mono-repo import of styles

### PR DESCRIPTION
**Pull request by jacob-ebey**
Closes: #

- [ ] Docs
- [ ] Tests

Testing Strategy:

### CODE REVIEW

**Review by mcansh — APPROVED** [review]

File: packages/remix-dev/compiler/plugins/cssFilePlugin.ts [hunk 1]
```diff
@@ -72,9 +72,11 @@ export function cssFilePlugin(options: {
         let entry = Object.keys(outputs).find((out) => outputs[out].entryPoint);
         invariant(entry, "entry point not found");
 
-        let normalizedEntry = normalizePathSlashes(entry);
```
Comment (pcattori) [inline packages/remix-dev/compiler/plugins/cssFilePlugin.ts:75]: If we want to guarantee that this is correct for all cases, I think its better to resolve the path relative to the project root directory specified in the Remix config. I think we can get that path by looking at `args.path` within this plugin.

Comment (jacob-ebey) [reply packages/remix-dev/compiler/plugins/cssFilePlugin.ts:75]: Ended up just passing through the config value so there is no guessing / accidental miss-slice of a bad path.

**Review by pcattori — APPROVED** [review]

File: packages/remix-dev/compiler/compileBrowser.ts [hunk 2]
```diff
@@ -73,7 +73,10 @@ const createEsbuildConfig = (
 
   let plugins: esbuild.Plugin[] = [
     deprecatedRemixPackagePlugin(options.onWarning),
-    cssFilePlugin(options),
+    cssFilePlugin({
+      mode: options.mode,
+      rootDir: config.rootDirectory,
```
Comment (machour) [inline packages/remix-dev/compiler/compileBrowser.ts:78]: `rootDirectory`

Comment (jacob-ebey) [reply packages/remix-dev/compiler/compileBrowser.ts]: Yeah, got distracted and pushed before I finished lolol

**Pull request merged**